src/modules/pages/components/RouteLayouts/UserNavigation.css

```diff
@@ -12,7 +12,7 @@
 .notificationsIcon {
   flex-grow: 0;
   margin-right: 15px;
-  padding: 7px 9px 9px;
+  padding: 7px 6px;
```

Comment:
This (or at least I assume it was this), broke the user notification dot:

![Screenshot from 2021-02-05 14-35-21](https://user-images.githubusercontent.com/1193222/107037049-36c38900-67c3-11eb-93c3-b784b3c15c8b.png)